packages/headless/config/database/index.ts

```diff
@@ -11,16 +11,16 @@ const {
 	MYSQL_DB_NAME,
 } = process.env;
 export const dbConfig = {
-	mongodb: {
-		URI: MONGODB_URI! || 'mongodb://localhost:27017/bs-commerce-dev',
-	},
-	mysql: {
-		username: MYSQL_DB_USER! || 'root',
-		password: MYSQL_DB_PASSWORD! || '',
-		database: MYSQL_DB_NAME! || 'bs-commerce-dev',
-		host: MYSQL_DB_HOST! || 'localhost',
-		port: parseInt(MYSQL_DB_PORT!) || 3306,
-		dialect: (MYSQL_DB_DIALECT as Dialect) || 'mysql',
-	},
-	db: DB || 'MONGO',
-};
+    mongodb: {
```

Comment:
why are the changes to this file?

Reply:
dont know how it got changed, maybe pulling from another branch or a test branch of mine did it